packages/components/src/Enumeration/Enumeration.component.js

```diff
@@ -52,6 +55,7 @@ Enumeration.propTypes = {
 	headerDefault: PropTypes.arrayOf(PropTypes.shape(headerPropTypes)).isRequired,
 	headerInput: PropTypes.arrayOf(PropTypes.shape(headerPropTypes)),
 	headerSelected: PropTypes.arrayOf(PropTypes.shape(headerPropTypes)),
+	id: PropTypes.string,
```

Comment:
Using multiple EnumrationComponent would break if there's no id right (duplicate id's in DOM)? Should we not make it required? 
(I'm not sure if there's a risk there would be more than one)

Reply:
Yes, and worst, there were already ids, generated from the index of the item in the list. So currently, the ids unicity breaks if you have multiple Enumeration in the page :/
I added this prop so all the ids are now based on the one passed by user.

I didn't want to make it required, because in the case you have only 1 instance, it isn't required. In fact like the List.

Reply:
I would have made the same remark with id. So, to avoid duplication id, should we use id generated by an uuid ?